tests/test_basic.py

```diff
@@ -5,11 +5,67 @@
 import unittest
 
 
+import crash_similarity
+
+
 class BasicTest(unittest.TestCase):
 
-    def test_pass(self):
+    # Train Model to be used in all tests
+    paths = ['crashsimilarity_data/test.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-09.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-08.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-07.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-06.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-05.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-04.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-03.json.gz']
+
+    corpus = crash_similarity.read_corpus(paths)
+
+    model = crash_similarity.train_model(corpus)
+    print('Model Trained with paths:\n{}'.format(paths))
+
+    def test_train_model(self):
         self.assertTrue(True)
 
+    # Test if at the number of zero distances are at least the number of different reports when pared against itself
```

Comment:
Can you rewrite this comment to make it clearer? Maybe also show an example of the stack traces in the `mozilla::testZeroCoherence` signature.